src/cli/localNode.ts

```diff
@@ -65,14 +71,18 @@ export class LocalNode {
       const str = data.toString();
       // Print local node output log
       // eslint-disable-next-line no-console
-      console.log(str);
+      if (this.showStdout) {
+        console.log(str);
```

Comment:
lol, moved down one line.

Let's just put the ignore lint at the top of the file so we don't have to worry about it anymore.